dev-lang/swift_lang/swift_lang-4.1.0~git.recipe

```diff
@@ -130,15 +130,19 @@ BUILD()
 	chmod +x ./swift/utils/build-script ./swift/utils/build-script-impl
 	chmod -R 755 ./swift/utils
 
-	export HOST_CC="/bin/clang"
+	export HOST_CC="/bin/clang_9"
 	export HOST_CXX="/bin/clang++"
 
 	# Build faster by building only for X86
-	python2 ./swift/utils/build-script -R -m \
+	python3.7 ./swift/utils/build-script -R -m \
```

Comment:
is it compatible with python3?

Reply:
Should be investigated, otherwise this PR could be closed, nothing is enabled so far ...

Reply:
I see upstream just recently switched to python3.

Reply:
I could try to switch to the latest version and see if the patchset still applies, but this is one build to start in the evening and hopefully will be done by the morning :)